site/src/components/WorkspacesTable/WorkspacesRow.tsx

```diff
@@ -58,20 +56,9 @@ export const WorkspacesRow: FC<{
       </TableCell>
 
       <TableCell>
-        <Link
-          component={RouterLink}
-          to={`/templates/${workspace.template_name}`}
-          className={styles.templateLink}
-          title={`Go to ${displayTemplateName} page`}
-          onClick={(e) => {
-            e.stopPropagation()
-          }}
-        >
-          <Stack direction="row" alignItems="center" spacing={1}>
-            <TemplateLinkIcon className={styles.templateLinkIcon} />
-            <span>{displayTemplateName}</span>
-          </Stack>
-        </Link>
+        <Stack direction="row" alignItems="center" spacing={1}>
+          <span>{displayTemplateName}</span>
+        </Stack>
```

Comment:
You can remove the Stack since there is a single element, you can also remove the span.